### DISCUSSION TITLE
Question about changing  temporal resolution in GCAM

### DISCUSSION THREAD

**Discussion by jiangyongye** [JGCRI/gcam-core#279] [Q&A] [answered]
Does anyone know how to change the default five years temporal resolution in GCAM to 1 year? I assumed that the reference scenario could be run under a one-year temporal resolution.
So I tried to change the constant MODEL_FUTURE_YEARS (2020, 2100, 5) in 'gcam-v6.0\input\gcamdata\R' to (2020, 2100, 1), but it just not work.

**Comment by jiangyongye** [comment]
I also tried to change the time-step to 1 in '<inter-year dummy-tag="1" time-step="5">2005</inter-year>' at modeltime.xml, and it still did not work, and when I ran it it just out a bunch of  'Skipping interpolation for xx in year 20xx due to mismatch.' errors

**Reply by ssmithClimate** [reply, marked as answer]
This needs to be done in the gcamdata processing system that generates the XMLs.

**Reply by Craven-Pirate** [reply]
Can you please elaborate on this? Unable to find the exact function to increase temporal resolution.
